Mu2eG4/src/Mu2eG4SteppingAction.cc

```diff
@@ -155,6 +175,13 @@ namespace mu2e {
       }
     }
 
+    //for killing low momentum photon conversions in materials 
+    if(gammaFilterAccepted_ < 0 || //if < 0 kill entire event without checking tracks
```

Comment:
Is this comment correct?

Reply:
Yes, once the photon is found not to convert to a high enough energy daughter, there's no point to continuing to simulate anything as this event will be filtered out. Is there a better way to implement this intent? I inherited the idea of using the SteppingAction to check for these conversions and kill events that don't pass the given threshold.

Reply:
I just saw the suggestion below about the special process killer, I'll start by looking into this.